docs/docs/chitchat-faqs.mdx

```diff
@@ -2,336 +2,72 @@
-one story for each of the different intents.
-
-The [ResponseSelector](components/selectors.mdx#responseselector) is designed to
-make it easier to handle conversation patterns like small talk and FAQ messages.
-When you use the `ResponseSelector`, you only need one story to handle all FAQs,
-instead of adding one story for each intent.
-
-:::note More details on the Response Selector
 
-This walk through shows you how to use the response selector. If you want to know
-more about how it works under the hood, head over to this [blog post](https://blog.rasa.com/response-retrieval-models/) and the
-[Retrieval Actions](./retrieval-actions.mdx) page.
-:::
+### 2. Defining Retrieval Actions and the `ResponseSelector`
 
-### 1. Creating intents
-
-People often ask Sara different questions surrounding the Rasa products, so let's
-start with three intents: `ask_channels`, `ask_languages`, and `ask_rasax`.
-
-Similar to the `MemoizationPolicy`, you'll need to define the intents and add some
-training data for them:
-
-```yaml-rasa title="data/nlu.yml"
-nlu:
-- intent: faq/ask_channels
-  examples: |
-    - What channels of communication does rasa support?
-    - what channels do you support?
-    - what chat channels does rasa uses
-    - channels supported by Rasa
-    - which messaging channels does rasa support?
-
-- intent: faq/ask_languages
-  examples: |
-    - what language does rasa support?
-    - which language do you support?
-    - which languages supports rasa
-    - can I use rasa also for another laguage?
-    - languages supported
-
-- intent: faq/ask_rasax
-  examples: |
-    - I want information about rasa x
-    - i want to learn more about Rasa X
-    - what is rasa x?
-    - Can you tell me about rasa x?
-    - Tell me about rasa x
-    - tell me what is rasa x
-```
-:::tip FAQ/ prefix
-It's important that these intents have a common prefix that is separated by a `/`.
-E.g. in the above example, all intents share the `faq/` prefix. This is necessary
-for the intents to be recognized by the `ResponseSelector`.
-
-:::
-
-### 2. Creating responses
-
-Next, you'll need to define the responses associated with these FAQs in a new
-file:
-
-```yaml-rasa title="data/responses.yml"
-responses:
-  faq/ask_channels:
-  - text: |
-      We have a comprehensive list of [supported connectors](https://rasa.com/docs/core/connectors/), but if
-      you don't see the one you're looking for, you can always create a custom connector by following
-      [this guide](./connectors/custom-connectors.mdx).
-
-  faq/ask_languages:
-  - text: "You can use Rasa to build assistants in any language you want!"
+Consider an example where you have 20 different FAQs. Although each question is represented as an individual intent, all FAQ intents are handled the same way in the dialogue. For each FAQ intent, the assistant **retrieves** the proper response depending on which question has been asked.
```

Comment:
"retrieves" doesn't really mean anything to the user. Is there a reason for the lengthy explanation here and below?

we had a very short explanation originally:
```
The [ResponseSelector](components/selectors.mdx#responseselector) is designed to
make it easier to handle conversation patterns like small talk and FAQ messages.
When you use the `ResponseSelector`, you only need one story to handle all FAQs,
instead of adding one story for each intent.
```
^ probably shouldn't be exactly that, but i don't think we need to do the whole "instead of writing 20 rules" thing